code/noRAM/app/src/main/java/com/example/noram/controller/EventManager.java

```diff
@@ -0,0 +1,79 @@
+package com.example.noram.controller;

```

Comment:
top comment

Reply:
Because the class is responsible of functionalities that goes on in the UI (and database), I think it would make more sense to test the class in UI test, when doing transitions to the EventInfo page.

Reply:
Sounds good!